Meta/run.sh

```diff
@@ -248,14 +248,31 @@ if [ -z "$SERENITY_DISABLE_GDB_SOCKET" ]; then
 fi
 
 if [ -z "$SERENITY_ETHERNET_DEVICE_TYPE" ]; then
-  SERENITY_ETHERNET_DEVICE_TYPE="e1000"
+    SERENITY_ETHERNET_DEVICE_TYPE="e1000"
+fi
+
+if [ -z "$SERENITY_HOST_TELNET_PORT" ]; then 
+    SERENITY_HOST_TELNET_PORT="8823"
+fi
+
+if [ -z "$SERENITY_HOST_WEBDRIVER_PORT" ]; then 
+    SERENITY_HOST_WEBDRIVER_PORT="8000"
+fi
+
+if [ -z "$SERENITY_HOST_SSH_PORT" ]; then 
+    SERENITY_HOST_WEBDRIVER_PORT="2222"
+fi
+
+# Could not find a service that uses this port by default
```

Comment:
this is pretty old and probably was used by andreas for development of tcp stack. i guess this is just an artefact to be removed, but could also be treated as a pre-configured port free to be used.

if not deleting, i guess you could name it in some generic way, e.g. `GENERIC`, `DUMMY` or something along those lines…

Reply:
Renamed it to `GENERIC`.